src/main/resources/js/rest.client.js

```diff
@@ -22,6 +22,14 @@ function getResponseAsReturnValue(url) {
 	return JSON.parse(xhr.response);
 }
 
+function postWithResponseAsReturnValue(url) {
```

Comment:
[Decision] I added this post method without nested callback function in order to get the response value for the progress bar [/Decision]
[Con] It freezes the page until the classification method is finished. This can take a while... [/Con]